lstchain/image/muon/muon_analysis.py

```diff
@@ -212,9 +212,8 @@ def analyze_muon_event(event_id, image, geom, equivalent_focal_length,
 
         pix_ringwidth_im = image * dist_ringwidth_mask
         idx_ringwidth = np.nonzero(pix_ringwidth_im)
```

Comment:
Why are indices needed here? The mask could be directly used and that would be faster?

Reply:
Right, replaced by:

> pix_ringwidth_im = image[dist_ringwidth_mask]
> muonintensityoutput.ring_pix_completeness =  npix_above_threshold(pix_ringwidth_im, tailcuts[0]) / len(pix_ringwidth_im)